src/View/Game/game.html.twig

```diff
@@ -0,0 +1,28 @@
+{% extends 'layout.html.twig' %}
+
+    {% block title %}
+        Stercket Battle
+    {% endblock %}
+
+    {% block link %}
+            <link rel="stylesheet" href="/assets/css/game.css">
+            <link rel="preconnect" href="https://fonts.gstatic.com">
+            <link href="https://fonts.googleapis.com/css2?family=Passero+One&display=swap" rel="stylesheet">
+    {% endblock %}
+    {% block content %}
+        {% block nav %}
+            {% block button %}
+                <button class = "button buttonHome"> Home </button>
+                <button class = "button buttonRest"> Rest </button>
+                <button class = "button buttonBattle"> Battle </button>
```

Comment:
Do you plan to have JS involved ?
If not this should probably be links or forms ot have a client/server interaction
And use CSS if you want them to looks like buttons